src/peerconnection.h

```diff
@@ -94,38 +94,30 @@ class PeerConnection
   };
 
   enum AsyncEventType {
-    CREATE_OFFER_SUCCESS = 0x1 << 0, // 1
-    CREATE_OFFER_ERROR = 0x1 << 1, // 2
-    CREATE_ANSWER_SUCCESS = 0x1 << 2, // 4
-    CREATE_ANSWER_ERROR = 0x1 << 3, // 8
-    SET_LOCAL_DESCRIPTION_SUCCESS = 0x1 << 4, // 16
-    SET_LOCAL_DESCRIPTION_ERROR = 0x1 << 5, // 32
-    SET_REMOTE_DESCRIPTION_SUCCESS = 0x1 << 6, // 64
-    SET_REMOTE_DESCRIPTION_ERROR = 0x1 << 7, // 128
-    ADD_ICE_CANDIDATE_SUCCESS = 0x1 << 8, // 256
-    ADD_ICE_CANDIDATE_ERROR = 0x1 << 9, // 512
-    NOTIFY_DATA_CHANNEL = 0x1 << 10, // 1024
-    NOTIFY_CONNECTION = 0x1 << 11, // 2048
-    NOTIFY_CLOSED_CONNECTION = 0x1 << 12, // 4096
-    ICE_CANDIDATE = 0x1 << 13, // 8192
-    SIGNALING_STATE_CHANGE = 0x1 << 14, // 16384
-    ICE_CONNECTION_STATE_CHANGE = 0x1 << 15, // 32768
-    ICE_GATHERING_STATE_CHANGE = 0x1 << 16, // 65536
-    NOTIFY_ADD_STREAM = 0x1 << 17, // 131072
-    NOTIFY_REMOVE_STREAM = 0x1 << 18, // 262144
-    GET_STATS_SUCCESS = 0x1 << 19, // 524288
-
-    ERROR_EVENT = CREATE_OFFER_ERROR | CREATE_ANSWER_ERROR |
-                  SET_LOCAL_DESCRIPTION_ERROR | SET_REMOTE_DESCRIPTION_ERROR |
-                  ADD_ICE_CANDIDATE_ERROR,
-    SDP_EVENT = CREATE_OFFER_SUCCESS | CREATE_ANSWER_SUCCESS,
-    VOID_EVENT = SET_LOCAL_DESCRIPTION_SUCCESS | SET_REMOTE_DESCRIPTION_SUCCESS |
-                 ADD_ICE_CANDIDATE_SUCCESS | GET_STATS_SUCCESS,
-    STATE_EVENT = SIGNALING_STATE_CHANGE | ICE_CONNECTION_STATE_CHANGE |
-                  ICE_GATHERING_STATE_CHANGE
+    CREATE_OFFER_SUCCESS = 1,
```

Comment:
I think this should be fine, as long as you unfolded the combined events from the old code.